packages/ic-asset-certification/src/asset_router.rs

```diff
@@ -133,19 +133,20 @@ fn encoding_str(maybe_encoding: Option<AssetEncoding>) -> Option<String> {
 }
 
 fn parse_range_header_str(str_value: &str) -> Result<RangeRequestValues, String> {
-    // expected format: `bytes <range-begin>-[<range-end>]`
-    let re = Regex::new(r"bytes=(\d+)-(\d*)").expect("internal: wrong RE");
-    let Some(caps) = re.captures(str_value) else {
-        return Err("malformed Range header".to_string());
+    // expected format: `bytes=<range-begin>-[<range-end>]`
+    let range_begin: usize = try_read!("bytes={}-", str_value.bytes())
```

Comment:
Does it work to use `try_scan!("bytes={}-{}", str_value.bytes())`?

Reply:
The problem is the optional `range-end` -- if it is missing, we should not complain.